MobiFlight/MidiBoard/MidiBoardManager.cs

```diff
@@ -29,40 +34,33 @@ public MidiBoardManager ()
         private List<string> CollectErrorsInInputDefinition(MidiBoardDefinition def)
         {
             List<string> errorMessages = new List<string>();
-            if (def.Inputs == null)
-                errorMessages.Add($"No input definitions.");
-            else
+
+            foreach (var item in def.Inputs)
             {
-                foreach (var item in def.Inputs)
-                {
-                    if (item.LabelIds.Length != item.MessageIds.Length)
-                        errorMessages.Add($"{item.Label}: Number of LabelIds unequal to number of MessageIds");
-                    if (!item.Label.Contains("%"))
-                        errorMessages.Add($"{item.Label}: Label does not contain % placeholder.");
-                }
+                if (item.LabelIds.Length != item.MessageIds.Count)
+                    errorMessages.Add($"{item.Label}: Number of LabelIds unequal to number of MessageIds");
             }
+
             return errorMessages;
         }
 
         private List<string> CollectErrorsInOutputDefinition(MidiBoardDefinition def)
         {
-            List<string> errorMessages = new List<string>();              
+            List<string> errorMessages = new List<string>();        
+            
             if (def.Outputs == null) return errorMessages;
             
             foreach (var item in def.Outputs)
             {
-                if (item.LabelIds.Length != item.MessageIds.Length)
+                if (item.LabelIds.Length != item.MessageIds.Count)
                     errorMessages.Add($"{item.Label}: Number of LabelIds unequal to number of MessageIds");
-                if (!item.Label.Contains("%"))
-                    errorMessages.Add($"{item.Label}: Label does not contain % placeholder.");
                 if (!string.IsNullOrEmpty(item.RelatedInputLabel))
                 {
-                    if (item.RelatedIds.Length != item.MessageIds.Length)
+                    if (item.RelatedIds.Length != item.MessageIds.Count)
                         errorMessages.Add($"{item.Label}: Number of RelatedIds unequal to number of MessageIds");
-                    if (!item.RelatedInputLabel.Contains("%"))
-                        errorMessages.Add($"{item.Label}: RelatedInputLabel does not contain % placeholder.");
```

Comment:
This is now enforced via the schema.

Reply:
Will these messages also become visible somewhere?

Reply:
Hmmm I think I missed logging these out after. Oops.

Reply:
I didn't, they get logged in CollectAndLogDefinitionErrors().